_sc2data/Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml

```diff
@@ -7388,6 +7383,240 @@
         <LaunchAttachQuery Methods="Weapon 4"/>
         <LaunchSiteOps HoldPosition="1"/>
     </CActorAction>
+    <CActorSound id="AP_ConsumptionImpactSound" parent="SoundOneShot">
+        <On Terms="Effect.AP_InfestorConsumption.Start" Send="Create"/>
+        <Sound value="Tosh_ConsumptionImpact"/>

```

Comment:
This doesn't exist in Liberty.SC2Mod. Use `AP_Tosh_ConsumptionImpact` that already exists